### PR TITLE
Completed html-tables exercise.

### PR DESCRIPTION
**Pull request by Edward-J-Kim**
Screenshot 2023-03-28 at 11 21 03 AM

### CODE REVIEW

File: html-tables/index.html [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+<head>
+  <title>html-tables</title>
+</head>
+
+<body>
+
+  <h1>Student Grade Table</h1>
+  <table>
+    <tr>
```
Comment (robgmerrill) [inline html-tables/index.html:12]: lines 12-16 should be wrapped in a `thead` element
```
<thead>
  <tr>
    <th>Student Name</th>
    <th>Course</th>
    <th>Grade</th></tr>
  </tr>
</thead>

**Review by robgmerrill — CHANGES_REQUESTED** [review]
Close. For best practice you should include the `thead` and `tbody` elements.

File: html-tables/index.html [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+  <th>Student Name</th>
+  <th>Course</th>
+  <th>Grade</th></tr>
+    <tr>
```
[the remark this answers is not in the document]
Comment (Edward-J-Kim) [reply html-tables/index.html]: Updated :)

**Review by robgmerrill — APPROVED** [review]
👍

**Pull request merged**